lcc_comchain_base/models/res_partner.py

```diff
@@ -55,8 +55,12 @@ def comchain_backend_accounts_data(self):
         if not backend_id:
             ## not present in wallet and not configured in general settings
             return []
-
-        data = {"type": backend_id, "accounts": []}
+        comchain_product = self.env.ref("lcc_comchain_base.product_product_comchain")
+        data = {
+            "type": backend_id,
+            "accounts": [],
+            "min_credit_amount": comchain_product.sale_min_qty,
```

Comment:
`sale_min_qty` won't be available if module `sale.order.min` is not installed. I would be to protect or check for existence of the field to let administrators of local currency decide if they need this or not.